src/Language/SystemVerilog/Parser/Parse.y

```diff
@@ -721,27 +721,42 @@ AssignOption :: { AssignOption }
 -- for ModuleItem, for now
 AssertionItem :: { AssertionItem }
   : ConcurrentAssertionItem { $1 }
+  | DeferredImmediateAssertionItem { $1 }
 
 -- for Stmt, for now
 ProceduralAssertionStatement :: { Assertion }
   : ConcurrentAssertionStatement { $1 }
-  | ImmediateAssertionStatement  { $1 }
+  | SimpleImmediateAssertionStatement  { $1 }
```

Comment:
Shouldn't this still be `ImmediateAssertionStatement`, as both varieties of immediate assertions are allowed in procedures?

Reply:
You are right. That's a mistake. Let me fix that.